src/scripts/main.js

```diff
@@ -358,3 +358,35 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+for (const person of people) {
+  const tr = document.createElement('tr');
+  const table = document.querySelector('table');
```

Comment:
Selecting the table element inside the loop can lead to performance inefficiencies. It's better to select the table element once before the loop starts and then append rows to it.